### DISCUSSION TITLE
https://plugin-components.vuejs.press/zh/guide/content/site-info.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#837] [Announcements]
# https://plugin-components.vuejs.press/zh/guide/content/site-info.html

站点信息组件，可用于友情链接或项目展示。 示例 属性 name 类型: string 必填: 是 站点名称 preview 类型: string 必填: 是 站点预览图，必须为绝对路径或完整 URL。 desc 类型: string 必填: 否 站点描述 logo 类型: string 必填: 否 站点图标 repo 类型: string 必填: 否 ...

https://plugin-components.vuejs.press/zh/guide/content/site-info.html

**Comment by kings1990** [comment]
如果 url 需要跳转到本站内的链接不是_blank 需要怎么设置
